test/test_plugin.py

```diff
@@ -0,0 +1,12 @@
+# Builtins
```

Comment:
Since plugins require extra setup I don't know if we should make tests for them.